### DISCUSSION TITLE
I receive an error with tifs

### DISCUSSION THREAD

**Discussion by valdier1** [ImageMagick/ImageMagick#5022] [Help] [answered]
I have a large number of tifs and am looking for a tool to convert them from grayscale to monochrome. I figure ImageMagick might be just what I need, but the problem is, anytime I try to convert, or even identify the tif's, I get this error:

![image](https://user-images.githubusercontent.com/13425341/162018549-bf295973-b7cb-42ca-a3ae-4287bd7bbb29.png)

This same tif opens in pretty much any other program, can be converted by other programs, etc. I am just not sure why this error occurs with these, specifically for ImageMagick, only.

Any suggestions would be greatly appreciated.

**Comment by snibgo** [comment]
The first two are warnings, not errors. The third is an error. For security reasons, IM has rules that files should comply with appropriate standards.

Ideally, you should contact whoever is responsible for the software that made that file to report the problem, and ask them to fix their software.

**Reply by valdier1** [reply]
That is unlikely to happen, because it's entire government filestores of tifs. I can't really imagine convincing the government counties to reformat all their tifs, or even just the offending ones, they just aren't that invested (and to get them to change it when IM is the only application that fails with them would be even harder).

I'm taking it there really isn't a work around for this then?

**Comment by snibgo** [comment]
Perhaps the tag can be ignored.

I suggest you give a link to a problem tif.

**Reply by valdier1** [reply]
Here is an example of the one shown in my error

[2763944-0001.zip](https://github.com/ImageMagick/ImageMagick/files/8429126/2763944-0001.zip)

**Reply by fmw42** [reply]
See https://imagemagick.org/script/defines.php



`-define tiff:ignore-tags=comma-separate-list-of-tag-IDs`    Allows one or more tag ID values to be ignored.


This works for me:

`magick -quiet -define tiff:ignore-tags=32934 2763944-0001.tif x.png`

![x](https://user-images.githubusercontent.com/13458218/162038065-59fd39f1-d7f5-4166-a5fb-02ac421bb0d3.png)

**Reply by valdier1** [reply]
Thank you, this perfectly does what I was trying to achieve, now I can work on the rest of the goal. I appreciate the help from everyone

**Comment by fmw42** [comment, marked as answer]
Put the image at the end:

`magick identify -define tiff:ignore-tags=32934,32931 2763944-0001.tif `
